ext/storage_sources/azure_store/azure_storage_source.cpp

```diff
@@ -645,7 +669,26 @@ int
 wiredtiger_extension_init(WT_CONNECTION *connection, WT_CONFIG_ARG *config)
 {
     azure_store *azure_storage = new azure_store;
+    WT_CONFIG_ITEM v;
+
     azure_storage->wt_api = connection->get_extension_api(connection);
+    int ret = azure_storage->wt_api->config_get(
+      azure_storage->wt_api, nullptr, config, "verbose.tiered", &v);
+
+    azure_storage->verbose = WT_VERBOSE_ERROR;
+    // Initialise logger for the storage source.
+    if (ret == 0 && v.val >= WT_VERBOSE_ERROR && v.val <= WT_VERBOSE_DEBUG_5) {
+        azure_storage->verbose = v.val;
+        Logger::SetLevel(wt_to_azure_verbosity_level(v.val));
```

Comment:
I would directly call Logger::SetLevel inside the `set_wt_verbosity_level` function. So that we can call set_wt_verbosity_level here

Reply:
amended in https://github.com/wiredtiger/wiredtiger/pull/8834/commits/1a85d68b179b622982fd10d057bfbeff6c9ad9a0